.github/workflows/ci.yml

```diff
@@ -44,42 +58,43 @@ jobs:
           # start xvfb in the background
           sudo /usr/bin/Xvfb $DISPLAY -screen 0 1280x1024x24 &
 
+      - name: Setup Brew cache (MacOS)
+        id: cache-brew
+        if: runner.os == 'macOS'
+        uses: actions/cache@v2
+        with:
+          path: |
+            /usr/local/Cellar/*
+            /usr/local/opt/*
```

Comment:
please stop doing this. Those are system directories managed by packaged by package managers; when there is an upgrade in github-actions runner itself(which is ~10 days) this will fail miserably. Also, this will be considerably large, for it to be in 5Gb.

Rather, we can download file which we needed from outside of brew and cache it, see previous comment.